django_react_proj/users-fe/src/components/Home/Home.js

```diff
@@ -39,7 +63,9 @@ const Home = ({ getUserForView, setCurrentDate, currentDate, users, getUsers })
             <Calendar fullscreen={false} onChange={(date) => setCurrentDate(date.toDate())} />
           </Modal>
         </Col>
+        <Col>{currentDate.toDateString().substring(3)}</Col>
```

Comment:
You can use format from moment if you have it imported anyway

Reply:
Done